deploy-service/teletraanservice/bin/server-no-db.yaml

```diff
@@ -0,0 +1,92 @@
+#
+# Dropwizard server configs. refer
+# http://www.dropwizard.io/0.9.1/docs/manual/configuration.html
+# for details.
+#
+server:
+  applicationConnectors:
+    - type: http
+      port: 8080
+  adminConnectors:
+    - type: http
+      port: 8081
+  requestLog:
+    appenders:
+      - type: file
+        threshold: ALL
+        archive: false
+        currentLogFilename: /tmp/teletraan/access.log
+        timeZone: UTC
+  registerDefaultExceptionMappers: false
+
+#
+# Logging settings. refer
+# http://www.dropwizard.io/0.9.1/docs/manual/configuration.html
+# for details.
+#
+logging:
+  level: INFO
+  loggers:
+    "com.pinterest.teletraan": DEBUG
+    "com.pinterest.deployservice": DEBUG
+  appenders:
+    - type: file
+      timeZone: UTC
+      threshold: DEBUG
+      archive: false
+      currentLogFilename: /tmp/teletraan/service.log
+    - type: console
+      threshold: ALL
+      timeZone: UTC
+      target: stdout
+
+
+#
+# Mysql connection config. By default, we use the real mysql as the backend.
+# Note: the embeded mysql is for demo or test purpose only. Use the official Mysql
+# installs for production usage.
+#
+db:
+  type: embedded
```

Comment:
This is really what I want to change - there is no better way I could find to not copy the whole server.yaml